rules/manager.go

```diff
@@ -37,9 +37,9 @@ import (
 const (
 	namespace = "prometheus"
 
-	ruleTypeLabel     = "rule_type"
-	alertingRuleType  = "alerting"
-	recordingRuleType = "recording"
+	RuleTypeLabel     = "rule_type"
```

Comment:
In general, having exported methods on non-exported structs is something that happens for various reasons. `String()` is an example of implementing an interface, which non-exported structs need to do, too.

I found myself quite often writing doc comments like "This is a helper method only used by nonExportedStruct internally." Perhaps a convention that capitalized methods of non-exported structs are meant to be used outside of the stuct (but within the package) would be helpful. Not sure, though, if a paradigm like that is common in the Go community.